### ISSUE TITLE
Start planning Knative Eventing WG documentation

### ISSUE THREAD

**Issue by Abd4llA** [kyma-project/kyma#6116]
**Description**

Prepare a vision for Knative Eventing documentation and create needed tickets on Knative docs repo. Align with the docs and eventing WG in Knative.

[This is an old doc ](https://docs.google.com/document/d/1nxNNzHoVoWulibSZBFGHo8EjVk47FK55NtfcBDto8w0/edit
)that describes the navigation layout and the goal was to structure the content in a way that any new user can start with Eventing asap without surfing through tonnes of documents. And then if needed can deep dive into more detailed documentation

**Comment by bszwarc** [comment]
Quick status update on this one:
I'm waiting on the response from Sam on the topic of Eventing documentation. What needs to be figured out is:
- The scope of the docu to be delivered - there is [this doc] (https://docs.google.com/document/d/1nxNNzHoVoWulibSZBFGHo8EjVk47FK55NtfcBDto8w0/edit#) but we don't know yet if it is still valid. 
- What is the current status of documentation - any work done so far? Where? Based on what (the doc mentioned above?)
- What is the plan - make it similar to Serving, better structured using the input that is already there + some improvements if necessary. Who can help with that, who is already involved? 
- How does it affect our Kyma documentation --> probably we will reuse some bits and pieces. 
Let's get the answers to these questions first and then estimate the work to be done.

**Comment by bszwarc** [comment]
After some discussions with Sam, I will probably take on the task of providing a "high-level" overview of Knative Eventing. 
The current proposal is [here](https://docs.google.com/spreadsheets/d/1L3EAeTX7RRdZhcQZeNmxwN4DCncuImCXA5A1_KQvwSk/edit#gid=413198505) ---> overview at line 186.
Next step is to have a meeting with an SME to give me a proper outline of Knative Eventing so I can start working on specific items.

**Comment by bszwarc** [comment]
Due to the focus on other Kyma-related documentation projects, this was put on hold. Can be closed for now.
